velox/docs/functions.rst

```diff
@@ -61,58 +61,61 @@ for :doc:`all <functions/coverage>` and :doc:`most used
-    :func:`degrees`                         :func:`negate`                          :func:`xxhash64`                                                                  
-    :func:`distinct_from`                   :func:`neq`                             :func:`year`                                                                      
-    :func:`divide`                          not                                     :func:`year_of_week`                                                              
-    :func:`dow`                             :func:`parse_datetime`                  :func:`yow`                                                                       
-    :func:`doy`                             :func:`pi`                              :func:`zip`                                                                       
+    :func:`abs`                             :func:`element_at`                      :func:`pow`                                 :func:`approx_distinct`
+    :func:`acos`                            :func:`empty_approx_set`                :func:`power`                               :func:`approx_most_frequent`
+    :func:`array_constructor`               :func:`eq`                              :func:`quarter`                             :func:`approx_percentile`
+    :func:`array_distinct`                  :func:`exp`                             :func:`radians`                             :func:`approx_set`
+    :func:`array_duplicates`                :func:`filter`                          :func:`rand`                                :func:`arbitrary`
+    :func:`array_except`                    :func:`floor`                           :func:`random`                              :func:`array_agg`
+    :func:`array_intersect`                 :func:`format_datetime`                 :func:`reduce`                              :func:`avg`
+    :func:`array_join`                      :func:`from_base`                       :func:`regexp_extract`                      :func:`bitwise_and_agg`
+    :func:`array_max`                       :func:`from_base64`                     :func:`regexp_extract_all`                  :func:`bitwise_or_agg`
+    :func:`array_min`                       :func:`from_hex`                        :func:`regexp_like`                         :func:`bool_and`
+    :func:`array_position`                  :func:`from_unixtime`                   :func:`regexp_replace`                      :func:`bool_or`
+    :func:`array_sort`                      :func:`greatest`                        :func:`replace`                             :func:`checksum`
+    :func:`array_sum`                       :func:`gt`                              :func:`reverse`                             :func:`corr`
+    :func:`array_sum_alt`                   :func:`gte`                             :func:`round`                               :func:`count`
```

Comment:
array_sum_alt is an alternative simple function implementation of array_sum. Do you think we should document this separately or exclude this from the functions coverage map?

Reply:
Where does array_sum_alt come from? I don't expect this to appear in the docs.

Reply:
I see it in velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp. I think it is a mistake. It would be nice to remove this in a separate PR.

```
template <typename OT, typename IT>
inline void registerArraySumFunction() {
  registerFunction<ArraySumFunction, OT, IT>({"array_sum_alt"});
}
```

Reply:
It was added in #1750 in ArrayFunctionsRegistration.cpp

Reply:
#2486 moved array_sum_alt to its appropriate place